br/pkg/task/common_test.go

```diff
@@ -151,13 +153,14 @@ func TestCheckCipherKey(t *testing.T) {
 }
 
 func TestCheckBackupAPIVersion(t *testing.T) {
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1, kvrpcpb.APIVersion_V1), true)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1TTL, kvrpcpb.APIVersion_V1TTL), true)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V2, kvrpcpb.APIVersion_V2), true)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1, kvrpcpb.APIVersion_V2), true)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1TTL, kvrpcpb.APIVersion_V2), true)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1, kvrpcpb.APIVersion_V1TTL), false)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V1TTL, kvrpcpb.APIVersion_V1), false)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V2, kvrpcpb.APIVersion_V1), false)
-	require.Equal(t, CheckBackupAPIVersion(kvrpcpb.APIVersion_V2, kvrpcpb.APIVersion_V1TTL), false)
+	featureGate := feature.NewFeatureGate(semver.New("6.1.0"))
```

Comment:
Suggest to add test cases when TiKV does not support api version conversion.

Reply:
Added.